### ISSUE TITLE
mcedit2 (Linux x64, Ubuntu 15.04) - SegFault Crash on Block Find/Replace

### ISSUE THREAD

**Issue by EvilSupahFly** [mcedit/mcedit2#163]
Version: commit d4f165 running on Ubuntu Linux 15.04 x64 - Map loads fine, and I can open "Find/Replace" but it crashes with a segfault error when I click the block selections for find/replace.

mcedit.log:
```
[INFO][main.py:164]:MCEdit2 version v2.0-tech-demo-994-gd4f1656 starting...
[INFO][acceleratesupport.py:12]:OpenGL_accelerate module loaded
[WARNING][arraydatatype.py:17]:Unable to load ArrayDatatype accelerator from OpenGL_accelerate
[WARNING][converters.py:122]:Unable to load converters accelerators (wrapper, arraydatatype) from OpenGL_accelerate
[WARNING][arrayhelpers.py:22]:Unable to load arrayhelpers accelerator from OpenGL_accelerate
[WARNING][numpymodule.py:28]:Unable to load numpy_formathandler accelerator from OpenGL_accelerate
[INFO][formathandler.py:54]:Unable to load registered array format handler numeric:
Traceback (most recent call last):
  File "/home/seann/.local/lib/python2.7/site-packages/PyOpenGL-3.0.1-py2.7.egg/OpenGL/arrays/formathandler.py", line 44, in loadPlugin  
    plugin_class = entrypoint.load()
  File "/home/seann/.local/lib/python2.7/site-packages/PyOpenGL-3.0.1-py2.7.egg/OpenGL/plugins.py", line 14, in load (self is a FormatHandler)
    return importByName( self.import_path )
  File "/home/seann/.local/lib/python2.7/site-packages/PyOpenGL-3.0.1-py2.7.egg/OpenGL/plugins.py", line 28, in importByName  
    module = __import__( ".".join(moduleName), {}, {}, moduleName)
  File "/home/seann/.local/lib/python2.7/site-packages/PyOpenGL-3.0.1-py2.7.egg/OpenGL/arrays/numeric.py", line 15, in <module>  
    raise ImportError( """No Numeric module present: %s"""%(err))
ImportError: No Numeric module present: No module named Numeric

[INFO][settings.py:100]:Loading app settings from /home/seann/mcedit_v2/MCEdit User Data/mcedit2.ini
[INFO][loadablechunks.py:17]:Making checkerboard texture...
[INFO][minecraftinstall.py:235]:Checking install at /home/seann/.minecraft
[INFO][minecraftinstall.py:242]:Found versions:
[u'1.6.4', u'1.7.4', u'1.8.8', u'1.8', u'1.7.2', u'1.7.10', u'1.8.7', u'1.8.1', u'1.7.9']
[INFO][minecraftinstall.py:235]:Checking install at /home/seann/.minecraft
[INFO][minecraftinstall.py:242]:Found versions:
[u'1.6.4', u'1.7.4', u'1.8.8', u'1.8', u'1.7.2', u'1.7.10', u'1.8.7', u'1.8.1', u'1.7.9']
[INFO][editorapp.py:467]:File not found: debug
[WARNING][editorapp.py:93]:UserFilesDirectory: /home/seann/mcedit_v2/MCEdit User Data
[INFO][editorapp.py:101]:Loaded translator.
[INFO][editorapp.py:110]:Loaded stylesheet.
[INFO][editorapp.py:122]:Loaded main window.
[INFO][editorapp.py:178]:Loaded panels.
[INFO][editorapp.py:228]:Loaded debug widgets.
[INFO][editorapp.py:279]:Loaded menus.
[INFO][worldlist.py:94]:Size hint: PySide.QtCore.QSize(145, 79)
[INFO][worldlist.py:95]:Size : PySide.QtCore.QSize(145, 100)
[INFO][worldlist.py:292]:Scanning /home/seann/.minecraft/saves for worlds...
[INFO][minecraftinstall.py:287]:Created ResourceLoader with search path:
/home/seann/.minecraft/versions/1.8.8/1.8.8.jar (6585891791e0d75dc54cfb547a866db0)
[INFO][qglcontext.py:52]:Obtained a GL context with this format:
Valid: True
Version: 1.3
Hardware Accelerated: True
Depth buffer: True, 24
Double buffer: True

Driver info:
GL_VERSION: 3.0 Mesa 11.0.2 (3, 0)
GL_VENDOR: 'X.Org'
GL_RENDERER: 'Gallium 0.4 on AMD TURKS (DRM 2.40.0, LLVM 3.6.0)'

[INFO][worldview.py:833]:LAYER Blocks VISIBLE True
[INFO][worldview.py:833]:LAYER MonsterLocations VISIBLE False
[INFO][worldview.py:833]:LAYER Items VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntities VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntityLocations VISIBLE False
[INFO][worldview.py:833]:LAYER CommandBlockColors VISIBLE True
[INFO][worldview.py:833]:LAYER CommandBlockLocations VISIBLE False
[INFO][worldview.py:833]:LAYER ItemFrames VISIBLE False
[INFO][worldview.py:833]:LAYER TileTicks VISIBLE False
[INFO][worldview.py:833]:LAYER ChunkSections VISIBLE False
[INFO][worldview.py:833]:LAYER HeightMap VISIBLE False
[INFO][worldview.py:185]:Changing MinimapWorldView(u'None') to dimension WorldEditorDimension(dimName='', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Link to -12_2'))
[INFO][chunkloader.py:140]:Added: client MinimapWorldView(u'Link to -12_2: ')
[INFO][worldlist.py:354]:Centering on spawn position.
[INFO][worldlist.py:369]:Switched world view
[INFO][editorapp.py:289]:Loaded world list.
[INFO][editorapp.py:308]:Loaded status bar.
[INFO][editorapp.py:603]:Updating recent files menu: (1) [u'/home/seann/.minecraft/saves/Seann']
[INFO][editorapp.py:618]:Adding Seann
[INFO][editorapp.py:315]:Loaded settings.
[WARNING][configureblocksdialog.py:119]:Failed to read definitions file defined_blocks.json
[INFO][editorapp.py:337]:Loaded app dialogs.
[INFO][editorapp.py:345]:Loading timer started
[INFO][editorapp.py:355]:Finding plugins
[INFO][plugins.py:204]:Loading plugins from /home/seann/mcedit_v2/src/plugins
[INFO][plugins.py:111]:Trying to load plugin from koch.py
[INFO][plugins.py:122]:Loaded koch.py (Koch Snowflake)
[INFO][plugins.py:111]:Trying to load plugin from hilbert.py
[INFO][plugins.py:122]:Loaded hilbert.py (Hilbert Curve)
[INFO][plugins.py:111]:Trying to load plugin from storagedrawers.py
[INFO][plugins.py:122]:Loaded storagedrawers.py (storagedrawers)
[INFO][plugins.py:111]:Trying to load plugin from city.py
[INFO][plugins.py:122]:Loaded city.py (GFX: City)
[INFO][editorapp.py:372]:Opening worlds from command line.
[INFO][worldlist.py:382]:Removing view from WorldListWidget
[INFO][minecraftinstall.py:287]:Created ResourceLoader with search path:
/home/seann/.minecraft/versions/1.8.8/1.8.8.jar (6585891791e0d75dc54cfb547a866db0)
[INFO][qglcontext.py:52]:Obtained a GL context with this format:
Valid: True
Version: 1.3
Hardware Accelerated: True
Depth buffer: True, 24
Double buffer: True

Driver info:
GL_VERSION: 3.0 Mesa 11.0.2 (3, 0)
GL_VENDOR: 'X.Org'
GL_RENDERER: 'Gallium 0.4 on AMD TURKS (DRM 2.40.0, LLVM 3.6.0)'

[INFO][worldview.py:833]:LAYER Blocks VISIBLE True
[INFO][worldview.py:833]:LAYER MonsterLocations VISIBLE False
[INFO][worldview.py:833]:LAYER Items VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntities VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntityLocations VISIBLE False
[INFO][worldview.py:833]:LAYER CommandBlockColors VISIBLE True
[INFO][worldview.py:833]:LAYER CommandBlockLocations VISIBLE False
[INFO][worldview.py:833]:LAYER ItemFrames VISIBLE False
[INFO][worldview.py:833]:LAYER TileTicks VISIBLE False
[INFO][worldview.py:833]:LAYER ChunkSections VISIBLE False
[INFO][worldview.py:833]:LAYER HeightMap VISIBLE False
[INFO][worldview.py:185]:Changing MinimapWorldView(u'None') to dimension WorldEditorDimension(dimName='', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Link to -12_2'))
[INFO][chunkloader.py:140]:Added: client MinimapWorldView(u'Link to -12_2: ')
[INFO][worldlist.py:354]:Centering on spawn position.
[INFO][worldlist.py:369]:Switched world view
[INFO][worldeditor.py:420]:Scanning for regions in /home/seann/.minecraft/saves/Link to -12_2...
[INFO][worldeditor.py:429]:Dim : Found 4096 chunks in 0.10 seconds.
[INFO][extensions.py:23]:OpenGL Version: 3.0 Mesa 11.0.2
[INFO][worldlist.py:382]:Removing view from WorldListWidget
[INFO][minecraftinstall.py:287]:Created ResourceLoader with search path:
/home/seann/.minecraft/versions/1.8.8/1.8.8.jar (6585891791e0d75dc54cfb547a866db0)
[INFO][editorsession.py:618]:Loading block models...
[INFO][editorsession.py:618]:Loading block models 0/1387
[INFO][editorsession.py:618]:Loading block models 100/1387
[INFO][editorsession.py:618]:Loading block models 200/1387
[INFO][editorsession.py:618]:Loading block models 300/1387
[INFO][editorsession.py:618]:Loading block models 400/1387
[INFO][editorsession.py:618]:Loading block models 500/1387
[INFO][editorsession.py:618]:Loading block models 600/1387
[INFO][editorsession.py:618]:Loading block models 700/1387
[INFO][editorsession.py:618]:Loading block models 800/1387
[INFO][editorsession.py:618]:Loading block models 900/1387
[INFO][editorsession.py:618]:Loading block models 1000/1387
[INFO][editorsession.py:618]:Loading block models 1100/1387
[INFO][editorsession.py:618]:Loading block models 1200/1387
[INFO][editorsession.py:618]:Loading block models 1300/1387
[INFO][editorsession.py:618]:Loading 7627 hidden blockState models...
[INFO][editorsession.py:618]:Found 1499 additional models for hidden states
[INFO][textureatlas.py:111]:Preloaded 360 textures for world Seann (513 kB)
[INFO][clone.py:262]:Begin clone: None
[INFO][__init__.py:244]:Loading BrushModeWidget setting: found fill at 1
[INFO][blockpicker.py:438]:Updating blocktype list widget (multiple=False) for <BlockTypeSet (Alpha)>
[INFO][blockpicker.py:524]:Updating button with [<BlockType (1:0) (Stone (minecraft:stone[variant=stone])>]
[INFO][__init__.py:204]:Updating brush cursor
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[INFO][__init__.py:204]:Updating brush cursor
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[INFO][__init__.py:204]:Updating brush cursor
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[INFO][__init__.py:204]:Updating brush cursor
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[INFO][__init__.py:204]:Updating brush cursor
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[INFO][__init__.py:204]:Updating brush cursor
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[INFO][__init__.py:204]:Updating brush cursor
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[INFO][blockpicker.py:438]:Updating blocktype list widget (multiple=False) for <BlockTypeSet (Alpha)>
[INFO][blockpicker.py:524]:Updating button with [<BlockType (1:0) (Stone (minecraft:stone[variant=stone])>]
[INFO][blockpicker.py:438]:Updating blocktype list widget (multiple=False) for <BlockTypeSet (Alpha)>
[INFO][blockpicker.py:524]:Updating button with [<BlockType (1:0) (Stone (minecraft:stone[variant=stone])>]
[INFO][generate.py:299]:Chose generator <koch.KochSnowflakePlugin object at 0x7f93da512878>
[INFO][editorsession.py:790]:Going to dimension 
[INFO][qglcontext.py:52]:Obtained a GL context with this format:
Valid: True
Version: 1.3
Hardware Accelerated: True
Depth buffer: True, 24
Double buffer: True

Driver info:
GL_VERSION: 3.0 Mesa 11.0.2 (3, 0)
GL_VENDOR: 'X.Org'
GL_RENDERER: 'Gallium 0.4 on AMD TURKS (DRM 2.40.0, LLVM 3.6.0)'

[INFO][worldview.py:833]:LAYER Blocks VISIBLE True
[INFO][worldview.py:833]:LAYER MonsterLocations VISIBLE False
[INFO][worldview.py:833]:LAYER Items VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntities VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntityLocations VISIBLE False
[INFO][worldview.py:833]:LAYER CommandBlockColors VISIBLE True
[INFO][worldview.py:833]:LAYER CommandBlockLocations VISIBLE False
[INFO][worldview.py:833]:LAYER ItemFrames VISIBLE False
[INFO][worldview.py:833]:LAYER TileTicks VISIBLE False
[INFO][worldview.py:833]:LAYER ChunkSections VISIBLE False
[INFO][worldview.py:833]:LAYER HeightMap VISIBLE False
[INFO][textureatlas.py:204]:Terrain atlas created for world Seann (513/545 kB)
[INFO][textureatlas.py:205]:Cooking quads for 2556 models...
[INFO][worldview.py:185]:Changing MinimapWorldView(u'None') to dimension WorldEditorDimension(dimName=u'', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Seann'))
[INFO][chunkloader.py:140]:Added: client MinimapWorldView(u'Seann: ')
[INFO][qglcontext.py:52]:Obtained a GL context with this format:
Valid: True
Version: 1.3
Hardware Accelerated: True
Depth buffer: True, 24
Double buffer: True

Driver info:
GL_VERSION: 3.0 Mesa 11.0.2 (3, 0)
GL_VENDOR: 'X.Org'
GL_RENDERER: 'Gallium 0.4 on AMD TURKS (DRM 2.40.0, LLVM 3.6.0)'

[INFO][worldview.py:833]:LAYER Blocks VISIBLE True
[INFO][worldview.py:833]:LAYER MonsterLocations VISIBLE False
[INFO][worldview.py:833]:LAYER Items VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntities VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntityLocations VISIBLE False
[INFO][worldview.py:833]:LAYER CommandBlockColors VISIBLE True
[INFO][worldview.py:833]:LAYER CommandBlockLocations VISIBLE False
[INFO][worldview.py:833]:LAYER ItemFrames VISIBLE False
[INFO][worldview.py:833]:LAYER TileTicks VISIBLE False
[INFO][worldview.py:833]:LAYER ChunkSections VISIBLE False
[INFO][worldview.py:833]:LAYER HeightMap VISIBLE False
[INFO][worldview.py:185]:Changing OverheadWorldView(u'None') to dimension WorldEditorDimension(dimName=u'', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Seann'))
[INFO][qglcontext.py:52]:Obtained a GL context with this format:
Valid: True
Version: 1.3
Hardware Accelerated: True
Depth buffer: True, 24
Double buffer: True

Driver info:
GL_VERSION: 3.0 Mesa 11.0.2 (3, 0)
GL_VENDOR: 'X.Org'
GL_RENDERER: 'Gallium 0.4 on AMD TURKS (DRM 2.40.0, LLVM 3.6.0)'

[INFO][worldview.py:833]:LAYER Blocks VISIBLE True
[INFO][worldview.py:833]:LAYER MonsterLocations VISIBLE False
[INFO][worldview.py:833]:LAYER Items VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntities VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntityLocations VISIBLE False
[INFO][worldview.py:833]:LAYER CommandBlockColors VISIBLE True
[INFO][worldview.py:833]:LAYER CommandBlockLocations VISIBLE False
[INFO][worldview.py:833]:LAYER ItemFrames VISIBLE False
[INFO][worldview.py:833]:LAYER TileTicks VISIBLE False
[INFO][worldview.py:833]:LAYER ChunkSections VISIBLE False
[INFO][worldview.py:833]:LAYER HeightMap VISIBLE False
[INFO][worldview.py:185]:Changing CutawayWorldView(u'None') to dimension WorldEditorDimension(dimName=u'', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Seann'))
[INFO][qglcontext.py:52]:Obtained a GL context with this format:
Valid: True
Version: 1.3
Hardware Accelerated: True
Depth buffer: True, 24
Double buffer: True

Driver info:
GL_VERSION: 3.0 Mesa 11.0.2 (3, 0)
GL_VENDOR: 'X.Org'
GL_RENDERER: 'Gallium 0.4 on AMD TURKS (DRM 2.40.0, LLVM 3.6.0)'

[INFO][worldview.py:833]:LAYER Blocks VISIBLE True
[INFO][worldview.py:833]:LAYER MonsterLocations VISIBLE False
[INFO][worldview.py:833]:LAYER Items VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntities VISIBLE False
[INFO][worldview.py:833]:LAYER TileEntityLocations VISIBLE False
[INFO][worldview.py:833]:LAYER CommandBlockColors VISIBLE True
[INFO][worldview.py:833]:LAYER CommandBlockLocations VISIBLE False
[INFO][worldview.py:833]:LAYER ItemFrames VISIBLE False
[INFO][worldview.py:833]:LAYER TileTicks VISIBLE False
[INFO][worldview.py:833]:LAYER ChunkSections VISIBLE False
[INFO][worldview.py:833]:LAYER HeightMap VISIBLE False
[INFO][worldview.py:185]:Changing CameraWorldView(u'None') to dimension WorldEditorDimension(dimName=u'', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Seann'))
[INFO][worldeditor.py:420]:Scanning for regions in /home/seann/.minecraft/saves/Seann...
[INFO][worldeditor.py:429]:Dim : Found 68227 chunks in 1.15 seconds.
[INFO][worldeditor.py:429]:Dim DIM1: Found 5638 chunks in 0.21 seconds.
[INFO][worldeditor.py:429]:Dim DIM-1: Found 6680 chunks in 0.15 seconds.
[INFO][editorsession.py:1342]:Going from OverheadWorldView(u'Seann: ') to CameraWorldView(u'Seann: '): Center was Vector(0, 0, 0)
[INFO][chunkloader.py:148]:Removed: client OverheadWorldView(u'Seann: ')
[INFO][chunkloader.py:140]:Added: client CameraWorldView(u'Seann: ')
[INFO][camera.py:309]:Camera: centering on Vector(0, 0, 0) (moving to Vector(-12.8171276411, 8.45236523481, -12.8171276411))
[INFO][editorsession.py:1350]:Center is now Vector(-12.8171276411, 8.45236523481, -12.8171276411)
[INFO][editorsession.py:1290]:Setting cursor SelectionCursor(visible=True, children=0) for tool <mcedit2.editortools.select.SelectionTool object at 0x7f93d23ada70> on view <mcedit2.worldview.camera.CameraWorldView object at 0x7f93da48d560>
[INFO][editorsession.py:440]:Update progressMax to 7, please.
[INFO][editorapp.py:529]:Adding session menus: [<PySide.QtGui.QMenu object at 0x7f93da89cb00>, <PySide.QtGui.QMenu object at 0x7f93da089680>, <PySide.QtGui.QMenu object at 0x7f93da089758>, <PySide.QtGui.QMenu object at 0x7f93da089488>]
[INFO][editorapp.py:603]:Updating recent files menu: (1) [u'/home/seann/.minecraft/saves/Seann']
[INFO][editorapp.py:618]:Adding Seann
[INFO][editorsession.py:971]:Setting view angle to single-player player's view in dimension DIM-1.
[INFO][editorsession.py:790]:Going to dimension DIM-1
[INFO][worldview.py:185]:Changing MinimapWorldView(u'Seann: ') to dimension WorldEditorDimension(dimName=u'DIM-1', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Seann'))
[INFO][worldview.py:185]:Changing OverheadWorldView(u'Seann: ') to dimension WorldEditorDimension(dimName=u'DIM-1', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Seann'))
[INFO][worldview.py:185]:Changing CutawayWorldView(u'Seann: ') to dimension WorldEditorDimension(dimName=u'DIM-1', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Seann'))
[INFO][worldview.py:185]:Changing CameraWorldView(u'Seann: ') to dimension WorldEditorDimension(dimName=u'DIM-1', adapter=AnvilWorldAdapter(u'/home/seann/.minecraft/saves/Seann'))
[INFO][chunkloader.py:140]:Added: client MinimapWorldView(u'Seann: DIM-1')
[INFO][chunkloader.py:140]:Added: client CameraWorldView(u'Seann: DIM-1')
[INFO][camera.py:309]:Camera: centering on Vector(-143.549810153, 102.384183305, 6.50320057251) (moving to Vector(-143.549810153, 102.384183305, 6.50320057251))
[INFO][extensions.py:83]:Chose alternate: glActiveTexture from glActiveTexture, glActiveTextureARB
[INFO][find_replace.py:188]:Row inserted
[INFO][find_replace.py:192]:Items created
[INFO][blockpicker.py:438]:Updating blocktype list widget (multiple=True) for <BlockTypeSet (Alpha)>
[INFO][blockpicker.py:524]:Updating button with [<BlockType (0:0) (tile.air.name (minecraft:air)>]
[INFO][find_replace.py:220]:Left button
[INFO][blockpicker.py:438]:Updating blocktype list widget (multiple=False) for <BlockTypeSet (Alpha)>
[INFO][blockpicker.py:524]:Updating button with [<BlockType (0:0) (tile.air.name (minecraft:air)>]
[INFO][find_replace.py:227]:Right button
[INFO][find_replace.py:234]:Done
```
Terminal Window:
```
seann@black-beast:~/mcedit_v2 $ mcedit2 debug
Logging level set
Logging to /home/seann/mcedit_v2/MCEdit User Data/mcedit.log
Setting up console handler
[WARNING][arraydatatype.py:17]:Unable to load ArrayDatatype accelerator from OpenGL_accelerate
[WARNING][converters.py:122]:Unable to load converters accelerators (wrapper, arraydatatype) from OpenGL_accelerate
[WARNING][arrayhelpers.py:22]:Unable to load arrayhelpers accelerator from OpenGL_accelerate
[WARNING][numpymodule.py:28]:Unable to load numpy_formathandler accelerator from OpenGL_accelerate
[WARNING][editorapp.py:93]:UserFilesDirectory: /home/seann/mcedit_v2/MCEdit User Data
[WARNING][configureblocksdialog.py:119]:Failed to read definitions file defined_blocks.json
Exception AttributeError: "'RevisionHistory' object has no attribute 'nodes'" in [WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
[WARNING][chunkupdate.py:490]:createVertexArrays: Block models not cooked, aborting.
Could not parse stylesheet of widget 0x5521d90
Could not parse stylesheet of widget 0x8878ab0
Could not parse stylesheet of widget 0x3f14330
Could not parse stylesheet of widget 0x54c1920
Could not parse stylesheet of widget 0x40ccd80
Could not parse stylesheet of widget 0x8b4ac20
Could not parse stylesheet of widget 0x4f79020
Could not parse stylesheet of widget 0x2f23870
Could not parse stylesheet of widget 0x69548d0
Could not parse stylesheet of widget 0x8a252d0
Could not parse stylesheet of widget 0x78d9ef0
Segmentation fault (core dumped)
```

**Comment by codewarrior0** [comment]
I can reproduce this.

Just to verify: When it segfaulted, my MCEdit2 appeared to freeze for two minutes before outputting "Segmentation fault". A minute later, the Ubuntu error reporter opened, saying "Sorry, the application python2.7 has stopped unexpectedly" and showing a list of detailed info, including the stacktrace. The top few lines of the stacktrace read something like this:

```
?? in libpyside-python2.7.so
PySide::SignalManager:;retrieveMetaObject
?? in PySide/QtGui.so
type_call(args=<[...]minecraft:stone[...], obj=<PySide.QtGui.QLabel>
PyObject_Call(kw=...)
```

Did you get a similar error report?

**Comment by codewarrior0** [comment]
The offending line is apparently

```
parentTypeLabel = QtGui.QLabel("")
```

:confused:

**Comment by codewarrior0** [comment]
Should be fixed in a5801f10341a40e598ec57e861362fe1e5bae667

Apparently creating widgets during `paint()` is verboten.

**Comment by EvilSupahFly** [comment]
Sorry it took so long to get back to you. Looking good now. Thanks!
